package.json

```diff
@@ -47,7 +47,6 @@
     "zod": "^3.20.2"
   },
   "devDependencies": {
-    "@chakra-ui/icons": "^2.0.17",
```

Comment:
i think we are still using chakra-ui/icons for the chevrons

Reply:
yeah, this is a duplicate dependency, "@chakra-ui/icons": "2.0.14", is declared in "dependencies"